### PR TITLE
[DOC-10386][DOC-11063] Update Plan a Standard Cluster, Cloud landing page

### PR DESCRIPTION
**Pull request by mdlinville**
[DOC-10386][DOC-11063] Update Plan a Standard Cluster, Cloud landing page

Based on https://docs.google.com/document/d/1l4_fHGtgA_x5YlD1GPb3wAqHIWg_ALT9W5S3YxOvxyA/edit, https://docs.google.com/document/d/1-GUUFb3JN4rrylB2wZtJy_GAuAQGfrNVA7A6KB7nqU0/edit, and related discussions

Drift from the draft:
- I did not use the examples with hypothetical charges. Instead, I tried to explain what they were illustrating without creating the potential for confusion if the actual bill is a different rate than the example.
- In the Plan Your Cluster page, I combined the sections on Estimating and Scaling costs into a single section
- Minor copyedit decisions

TODO:
- Figure out where to deal with the ratios between Standard and Advanced
- Formatting (width) of updated Cloud landing page

Out of scope for this PR:
- Copyedit and updates to Plan a Basic Cluster, Plan an Advanced. cluster

Open questions:
- In the draft, there's a contridiction in how Basic storage costs are calculated. Is it a static rate or does it vary based on Cloud provider and region?

### Previews
Major:
[src/current/cockroachcloud/index.md](https://deploy-preview-18930--cockroachdb-docs.netlify.app/docs/cockroachcloud/index.html)
[src/current/cockroachcloud/plan-your-cluster.md](https://deploy-preview-18930--cockroachdb-docs.netlify.app/docs/cockroachcloud/plan-your-cluster.html)

Minor:
[src/current/cockroachcloud/basic-cluster-management.md](https://deploy-preview-18930--cockroachdb-docs.netlify.app/docs/cockroachcloud/basic-cluster-management.html)
[src/current/cockroachcloud/cluster-management.md](https://deploy-preview-18930--cockroachdb-docs.netlify.app/docs/cockroachcloud/cluster-management.html)

### CODE REVIEW

**Comment by mdlinville** [comment]
Not sure what happened with that last commit, now undone and fixed.

File: src/current/cockroachcloud/index.md [hunk 1]
```diff
@@ -1,152 +1,247 @@
...
+
+{{site.data.alerts.callout_info}}
+There is no charge for backup storage during [Usage-based backup metrics Preview]({% link {{ site.current_cloud_version}}/essential-metrics-self-hosted.md %}#backup-and-restore)
+{{site.data.alerts.end}}
+
+<section class="filter-content" markdown="1" data-scope="basic">
+
+Storage for CockroachDB {{ site.data.products.basic }} is automatically provisioned on demand, eliminating the need for manual management. Costs are based on actual usage and adjust dynamically as data is added or removed, starting from zero.
+
+Storage is billed hourly, with [rates](https://www.cockroachlabs.com/pricing) depending on the cloud provider and the region where the cluster is hosted.{% comment %}TODO: Resolve this contradiction. https://cockroachlabs.slack.com/archives/C05KHQULKFB/p1726955009576009{% endcomment %}
```
Comment (mdlinville) [inline src/current/cockroachcloud/index.md:136]:
```suggestion
Storage is billed hourly. Refer to [Pricing](https://www.cockroachlabs.com/pricing).
```

File: src/current/cockroachcloud/basic-cluster-management.md [hunk 2]
```diff
@@ -35,11 +35,13 @@ The monthly cost estimate is calculated using simple extrapolation that assumes
 
 ## Edit cluster capacity
 
-On the **Overview** page, you can edit your resource limits. Changes apply to the current and future billing cycles. For more details, refer to [Plan a CockroachDB {{ site.data.products.basic }} cluster]({% link cockroachcloud/plan-your-cluster-basic.md %}).
+The price-performance characteristics of CockroachDB {{ site.data.products.basic }} vary significantly depending on the workload. It can be difficult to estimate a workload's compute requirements in advance. To increase or decrease a cluster's capacity:
```
Comment (marksoper) [inline src/current/cockroachcloud/basic-cluster-management.md:38]: @mdlinville there is a mismatch here. Cluster capacity is relevant only to Standard, not Basic.

Comment (marksoper) [reply src/current/cockroachcloud/basic-cluster-management.md:38]: Basic is on-demand so there's no capacity setting.

File: src/current/cockroachcloud/basic-cluster-management.md [hunk 3]
```diff
@@ -35,11 +35,13 @@ The monthly cost estimate is calculated using simple extrapolation that assumes
...
 
 1. In the **Capacity used this month** section of your cluster's [**Overview** page]({% link cockroachcloud/cluster-overview-page.md %}), click **Update capacity**.
 
-1. On the **Capacity** page, set the configure the new resource limits. In the sidebar, click **Update cluster**.
+1. On the **Capacity** page, edit the resource limits. In the sidebar, click **Update cluster**.
```
Comment (marksoper) [inline src/current/cockroachcloud/basic-cluster-management.md:42]: @mdlinville this part, editing resource limits, looks right for Basic

File: src/current/cockroachcloud/basic-cluster-management.md (hunk 2 above)
Comment (mdlinville) [inline src/current/cockroachcloud/basic-cluster-management.md:38]:
```suggestion
The price-performance characteristics of CockroachDB {{ site.data.products.basic }} vary significantly depending on the workload. It can be difficult to estimate a workload's compute requirements in advance. With {{ site.data.products.basic }}, by default compute and storage scale with your cluster's needs, but you can limit a cluster's resource usage:
```

File: src/current/cockroachcloud/plan-your-cluster.md [hunk 4]
```diff
@@ -8,163 +8,99 @@ cloud: true
...
 
-The pricing per-vCPUs does not depend in any way on the workload. Standard vCPUs are priced at 60% of the price of Advanced vCPUs, with some slight variations depending on the cloud provider and region. Since vCPUs are equivalent between Standard and Advanced on average, both plans require the same number of vCPUs on average, so total cluster cost for Standard is 60% of Advanced *on average*.
+- **Smaller workloads**: CockroachDB {{ site.data.products.standard }} is well suited for workloads that require 12 or fewer vCPUs. By contrast, 12 vCPUs (3 nodes x 4 vCPUs) is the minimum configuration for CockroachDB {{ site.data.products.advanced }}.
 
-But workloads may vary significantly from the average, so the estimation challenge is one of predicting the actual number of vCPUs a workload may require, regardless of whether you’re considering Standard or Advanced.
+  CockroachDB {{ site.data.products.standard }} is not recommended for:
+    - Analytical workloads that involve a large quantity of large, complex reads
+    - OLAP or hybrid OLTP/OLAP applications
```
Comment (mikeCRL) [inline src/current/cockroachcloud/plan-your-cluster.md:64]:
```suggestion
- **Analytical workloads that involve many large, complex reads**
- **OLAP or hybrid OLTP/OLAP applications**
```

File: src/current/cockroachcloud/plan-your-cluster.md [hunk 5]
```diff
@@ -8,163 +8,99 @@ cloud: true
...
 
-### Comparing total cluster vCPU capacity cost between Standard and Advanced
+- **Asymmetric multi-region workloads**: Often, multi-region deployments are asymmetric in terms of CPU utilization across regions. For example, in a three-region cluster, two of the three regions may require more capacity than the third, or vice versa. In CockroachDB {{ site.data.products.standard }}, a cluster's compute capacity is available in all regions.
 
-Given that Standard vCPUs are priced at 60% of Advanced vCPUs, estimating the cost of a workload on Standard as compared to Advanced means estimating the difference in number of vCPUs required between the two plans. For some workloads, Standard requires higher vCPU capacity, for others Advanced, depending on the properties of the workload and how the different underlying CockroachDB deployment architectures provide resources to that workload.
+  For workloads that require symmetric capacity per region or where a given region must maintain a given compute capacity, CockroachDB {{ site.data.products.advanced }} may be a better option.
```
Comment (mikeCRL) [inline src/current/cockroachcloud/plan-your-cluster.md:68]: 1. symmetric -> asymmetric?
2. This isn't showing up as a second paragraph of the same bullet. I suspect the blank line after is to blame; that the whole list should have no blank spaces except for the space *above* this line. But I'm not sure.

File: src/current/cockroachcloud/plan-your-cluster.md [hunk 6]
```diff
@@ -8,163 +8,99 @@ cloud: true
...
-Outcome: For 75% of all all tested workloads, the required Standard total cluster cost (in $ per month) is within 40% \- 70% of the total Advanced cluster cost (in $ per month).
+For more details to help you compare CockroachDB {{ site.data.products.cloud }} cluster plans, refer to [CockroachDB {{ site.data.products.cloud }} Overview]({% link cockroachcloud/index.md %}).
 
-The following histogram provides more detail on the test outcomes.  
-![][image3]
+## Storage pricing
 
-## Examples and Benchmarks for vCPU Capacity
+Storage capacity for a CockroachDB {{ site.data.products.standard }} cluster is [priced]({% link cockroachcloud/billing-management.md %}) on demand in units of GiB-month. Storage prices vary across providers and [regions]({% link cockroachcloud/regions.md %}). Storage is charged based on the logical amount of data in the database.
+- **3 replicas included**: - CockroachDB {{ site.data.products.standard }} maintains at minimum three replicas of your data at no additional cost.
```
Comment (mikeCRL) [inline src/current/cockroachcloud/plan-your-cluster.md:79]: Similar bullet fixes needed in this section
Screenshot 2024-09-25 at 1 10 59 AM

File: src/current/cockroachcloud/plan-your-cluster.md [hunk 7]
```diff
@@ -8,163 +8,99 @@ cloud: true
...
-| :---- | :---- | :---- | :---- | :---- | :---- | :---- |
-| TPC-C (150 warehouses) | 4.98 | 4.86 | 1.03 | $378 | $615 | 39%  |
-| TPC-C (300 warehouses) | 6.51 | 6.51 | 1.00 | $495 | $825 | 40% |
-|  |  |  |  |  |  |  |
+By default, CockroachDB {{ site.data.products.cloud }} maintains three replicas for a single-region cluster, spread across three different availability zones within the region. You are not charged to store these replicas.
 
+For example, if a single-region cluster is deployed in `us-central-1` and contains 500 GiB of data. CockroachDB {{ site.data.products.cloud }} stores three replicas, for a total of 1500 GiB of data. However, you are billed only for 500 GiB of storage.
 
-TODO - remove original table
+If you add a fourth replicas, the cluster consumes 2000 GiB of storage (500 x 4), but you are billed only for 1000 GiB (500 for the data and 500 for the fourth replica).
```
Comment (mikeCRL) [inline src/current/cockroachcloud/plan-your-cluster.md:91]:
```suggestion
If you add a fourth replica, the cluster consumes 2000 GiB of storage (500 x 4), but you are billed only for 1000 GiB (500 for the data and 500 for the fourth replica).
```

File: src/current/cockroachcloud/plan-your-cluster.md [hunk 8]
```diff
@@ -8,163 +8,99 @@ cloud: true
...
-| TPC-C (600 warehouses) | 9.45 | 10.90 | $69.21 |
-| TPC-E (500 customers) | 4.20 | 3.85 | $54.97 |
-| TPC-H (query \#1) | 10.64 | 18.81 | $106.03 |
-| TPC-H (query \#2) | 10.41 | 10.42 | $60.07 |
-| TPC-H (query \#11) | 11.02 | 12.83 | $69.87 |
-| TPC-H (query \#13) | 5.23 | 4.43 | $50.81 |
+### Multi-region example
 
-This table is \[WIP\]
+For multi-region clusters, we recommend 5 or more replicas are typically maintained across availability zones and regions for resilience against both zone and region failure. The first three replicas are included in the base storage price, while additional replicas are priced based on the ratio of additional regions to the initial three.
```
Comment (mikeCRL) [inline src/current/cockroachcloud/plan-your-cluster.md:95]:
```suggestion
For multi-region clusters, we recommend that 5 or more replicas are typically maintained across availability zones and regions for resilience against both zone and region failure. The first three replicas are included in the base storage price, while additional replicas are priced based on the ratio of additional regions to the initial three.
```

also:
* "we typically recommend" or "typically maintained" (i.e. you usually have this, or for most of your tables)?
* starting with the last sentence, I find this section/example hard to follow. May be worth a follow-up post launch to find a clearer way of explaining.

**Review by mikeCRL — COMMENTED** [review]
Reviewed plan your cluster + feature availability. 

TBD on how we handle index.md; as we discussed, I'd like to find a way to split it up and bring back the separate Cloud Costs page.

File: src/current/cockroachcloud/plan-your-cluster.md [hunk 9]
```diff
@@ -8,163 +8,99 @@ cloud: true
...
+- **Additional capacity buffer**: To accommodate unanticipated load and mitigate the risk of application degradation if the cluster's capacity is exceeded, we recommend that you plan for additional capacity beyond your initial estimate. A 40% buffer is a good starting point, but more may be needed if the workload is unpredictable or highly sensitive to query latency. Depending on the nature of the workload, such as when demand is seasonal, you can plan to scale the cluster's capacity temporarily to address the workload's requirements.
+  - **Exceeding capacity limit**: CockroachDB {{ site.data.products.standard }} enforces the cluster's provisioned capacity, limiting application throughput if the workload exceeds the cluster's provisioned capacity. This can lead to increased query latency and degraded application performance.
+  - **Notifications to prevent overrun**: To help mitigate against the risk of exceeding a cluster's capacity, email and CockroachDB {{ site.data.products.cloud }} Console notifications will occur when a cluster workload exceeds 90% and 90% of its provisioned capacity.
```
Comment (mdlinville) [inline src/current/cockroachcloud/plan-your-cluster.md:30]:
```suggestion
  - **Notifications to prevent overrun**: To help mitigate against the risk of exceeding a cluster's capacity, email and CockroachDB {{ site.data.products.cloud }} Console notifications will occur when a cluster workload exceeds 70% and 90% of its provisioned capacity.
```

File: src/current/cockroachcloud/plan-your-cluster.md (hunk 5 above)
Comment (mdlinville) [inline src/current/cockroachcloud/plan-your-cluster.md:68]:
```suggestion
    For workloads that require asymmetric capacity per region or where a given region must maintain a given compute capacity, CockroachDB {{ site.data.products.advanced }} may be a better option.
```

File: src/current/cockroachcloud/plan-your-cluster.md (hunk 6 above)
Comment (mdlinville) [inline src/current/cockroachcloud/plan-your-cluster.md:79]:
```suggestion
  - **3 replicas included**: CockroachDB {{ site.data.products.standard }} maintains at minimum three replicas of your data at no additional cost.
```

File: src/current/cockroachcloud/plan-your-cluster.md [hunk 10]
```diff
@@ -8,163 +8,99 @@ cloud: true
...
 
-## Examples and Benchmarks for vCPU Capacity
+Storage capacity for a CockroachDB {{ site.data.products.standard }} cluster is [priced]({% link cockroachcloud/billing-management.md %}) on demand in units of GiB-month. Storage prices vary across providers and [regions]({% link cockroachcloud/regions.md %}). Storage is charged based on the logical amount of data in the database.
+- **3 replicas included**: - CockroachDB {{ site.data.products.standard }} maintains at minimum three replicas of your data at no additional cost.
 
-Comparing relative CPU utilization for several benchmarks. Understanding the results below:
+  For a cluster with more than three replicas, storage for each additional replica is priced at the storage price for the region.
 
-### TPC-C
+- **Multi-region storage**: In a multi-region cluster, storage prices vary by region. The data in each region is priced at that region's storage rate.
```
Comment (mdlinville) [inline src/current/cockroachcloud/plan-your-cluster.md:83]:
```suggestion
- **Multi-region storage**: In a multi-region cluster, storage prices vary by region. The data in each region is priced at that region's storage rate.
```
```suggestion
    - **Multi-region storage**: In a multi-region cluster, storage prices vary by region. The data in each region is priced at that region's storage rate.
```

Comment (mdlinville) [inline src/current/cockroachcloud/plan-your-cluster.md:83]:
```suggestion
    - **Multi-region storage**: In a multi-region cluster, storage prices vary by region. The data in each region is priced at that region's storage rate.
```

File: src/current/cockroachcloud/plan-your-cluster.md (hunk 8 above)
Comment (mdlinville) [reply src/current/cockroachcloud/plan-your-cluster.md:95]: Updated your suggested edit, thanks. For your second point, it is confusing but it is explained in the next paragraph. You are charged N/3, so if you have 5 you are charged 2/3 the rate. If you have 6 you are charged 100% the rate. LMK if you think of a better way to put it.
